src/jdom/scheduler.ts

```diff
@@ -1,3 +1,6 @@
+import { assert } from "./jacdac-jdom"
```

Comment:
since this scheduler is bringing a new dependency (heap), we should move it out of the jdom library and in the test folder. It's generally not used outside of unit testing.

Reply:
Done (I think?).